src/test/java/edu/oakland/test/database/DatabaseCommManagerStub.java

```diff
@@ -0,0 +1,31 @@
+package edu.oakland.test.database;
+
+import edu.oakland.helper.admin.LocationDataPoint;
+import edu.oakland.helper.admin.TrackData;
+import edu.oakland.production.database.DatabaseCommManager;
+
+public class DatabaseCommManagerStub implements DatabaseCommManager {
+  int rfid = 10;
```

Comment:
Let's make sure that this is private!
```suggestion
  private int rfid;
```